Tests/test_DSSP_tool.py

```diff
@@ -46,20 +54,23 @@ def test_dssp(self):
         self.assertEqual(len(dssp), 130)
 
     def test_dssp_with_mmcif_file(self):
-        """Test DSSP generationg from MMCIF"""
-        p = MMCIFParser()
-        pdbfile = "PDB/2BEG.cif"
-        model = p.get_structure("2BEG", pdbfile)[0]
-        dssp = DSSP(model, pdbfile)
-        self.assertEqual(len(dssp), 130)
+        """Test DSSP generation from MMCIF"""
+        # Only run tests if DSSP version installed supports mmcif
+        if StrictVersion(dssp_version) >= StrictVersion('2.2.0'):
```

Comment:
Does it make sense to put this if statement outside the ``def``? i.e. Only define this test (and the one following) if using a recent DSSP?